google-spark-operator/operator/base/deployment.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: spark-operator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/instance: spark-operator
+      app.kubernetes.io/name: spark-operator
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      annotations:
+        prometheus.io/path: /metrics
+        prometheus.io/port: "10254"
+        prometheus.io/scrape: "true"
+      labels:
+        app.kubernetes.io/instance: spark-operator
+        app.kubernetes.io/name: spark-operator
+    spec:
+      containers:
+        - name: sparkoperator
+          image: gcr.io/spark-operator/spark-operator:v1beta2-1.2.3-3.1.1
```

Comment:
Can we move to our internal quay.io repo?

Reply:
I don't have access to the quay repo to do it. But as soon as the image is copied there I can make the changes.

Reply:
@guimou I've added you to the opendatahub quay org.  you should be able to push away.

Reply:
Thanks @crobby! Is that OK if I name it google-spark-operator to differentiate if from radanalytics? Other suggested name?

Reply:
That name sounds fine to me.

Reply:
Changed for ODH quay repo